experiment/compatibility-versions/e2e-k8s-compatibility-versions.sh

```diff
@@ -68,12 +68,10 @@ signal_handler() {
 }
 trap signal_handler INT TERM
 
-# build kubectl, kubernetes e2e test binaries, and ginkgo
-build() {
-  GINKGO_SRC_DIR="vendor/github.com/onsi/ginkgo/v2/ginkgo"
-
+# build kubectl
+build_kubectl() {
   # make sure we have e2e requirements
-  make all WHAT="cmd/kubectl test/e2e/e2e.test ${GINKGO_SRC_DIR}"
+  make all WHAT="cmd/kubectl"
```

Comment:
We should maybe also build the client in the N-... version? e2e.test will invoke it.

(also, TODO is still to fetch these from dl.k8s.io and invoke the fetched binaries more directly)

Reply:
I've changed this to no longer split this up into essentially keeping the "build" function the same but moving it to be done for the prev repo.  Now kubectl, e2e.test, and ginkgo are built from the prev version